domain-models/samples/AU128.gas.ttl

```diff
@@ -1,42 +1,34 @@
 @prefix dct: <http://purl.org/dc/terms/> .
 @prefix foaf: <http://xmlns.com/foaf/0.1/> .
 @prefix foiType: <http://pid.geoscience.gov.au/def/voc/featureofinteresttype/> .
+@prefix gas: <http://pid.geoscience.gov.au/def/ga-samples> .
 @prefix geo: <http://www.opengis.net/ont/geosparql#> .
 @prefix org: <http://www.w3.org/ns/org#> .
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
-@prefix samfl: <http://def.seegrid.csiro.au/ontology/om/sam-lite#> .
 
 <http://pid.geoscience.gov.au/org/ga/geoscienceaustralia> a org:Organization ;
     foaf:name "Geoscience Australia" .
 
 <http://pid.geoscience.gov.au/sample/AU128> 
-    a samfl:Specimen ;
+    a gas:Specimen ;
```

Comment:
This is not wrong but Simon Cox suggests `MaterialSample`, like https://w3id.org/tern/ontologies/tern/MaterialSample, so let's rename the class to that and retain a `owl:equivalentClass` link to `samfl:Specimen`